app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
- [ ] As you're using the email confirmation feature from Devise, please make sure to render this page correctly to have the users confirm their accounts directly from the app not by copying the link from the terminal because first the app is deployed and links can't be retrieved as in development mode, and second, not all users are familiar with this technical stuff. 🙏